### DISCUSSION TITLE
Can I Audit AsyncBackgroundJob

### DISCUSSION THREAD

**Discussion by gwilczura** [abpframework/abp#12539] [Q&A] [answered]
Hi.

Is it possible to enable audit logging for Background Jobs?
Neither `AuditedAttribute` nor `IAuditingEnabled` applied to `AsyncBackgroundJob` seem to be doing anything :)

Thx.

**Comment by gwilczura** [comment, marked as answer]
Ok. I might have just solved my own problem. I'm working on version 4.2.
`AuditingInterceptor` is significantly different than in the latest one.
Thesre is no `ProcessWithNewAuditingScopeAsync` in 4.2 - so since there is no `AuditingScope` on the `AsyncBackgroundJob` so nothing gets audited.

**Reply by johnhbarrett** [reply]
that's a pity - would be useful...
